src/go/transform-sdk/internal/cache/cache.go

```diff
@@ -0,0 +1,60 @@
+package cache
+
+import "math"
+
+type (
+	entry[V any] struct {
+		value           V
+		insertionNumber int
+	}
+	// A cache that evicts based on number of entries
```

Comment:
It appears to use FIFO eviction, correct?

Reply:
Correct, updated the documentation.